src/open_inwoner/templates/pages/contactmoment/detail.html

```diff
@@ -5,7 +5,7 @@
     {% if contactmoment %}
         {% render_grid %}
             {% render_column span=12 %}
-                <h1 class="utrecht-heading-1" id="title">{{ contactmoment.onderwerp }}</h1>
+                <h1 class="utrecht-heading-1" id="title">{{ contactmoment.subject }}</h1>
```

Comment:
Our API dataclasses have Dutch expressions for the fields (`contactmoment.identificatie`, `contactmoment.onderwerp` etc.). We should keep these consistent, so either we stick to Dutch (preference) or move the dataclasses to English as well. Unless there is some reason for and benefit from this divergence.